search-improvements.rst

```diff
@@ -0,0 +1,51 @@
+.. post:: August 14, 2018
+   :tags: gsoc, search, feature
+   :author: Safwan
+   :location: DHA
+
+Search Improvements
+====================
+
```

Comment:
A very short paragraph here explaining the biggest goals of the search upgrade would be good, I think. You want to get people interested so they read further.